Apps/frontend/src/views/ChannelConfigControlPanel.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import ChannelConfigPopup from "./ChannelConfigPopup.svelte";
+  import { LABEL_BUTTON_MANAGE_CHANNEL_CONFIG } from "../labels";
+  import { presetPopupOpen } from "../stores";
+  import { fly } from "svelte/transition";
+  import { getAllChannelConfig } from "../rest/ChannelConfigController";
+  import { availableChannelConfigPresets } from "../stores";
+
+  $: panelHeight = 0;
+  export let waveElement;
+  export let indicatorElement;
+
+  async function loadAllChannelConfigPresets() {
+    var response = await getAllChannelConfig();
+    if (response !== undefined) availableChannelConfigPresets.set(response);
+  }
+
+  function showPopup() {
+    $presetPopupOpen = true;
+    loadAllChannelConfigPresets();
+  }
+</script>
+
+{#if $presetPopupOpen}
```

Comment:
The popup should probably be the nav, not be inside the nav.

Reply:
Not sure, what to change, I recreated the logic friom the ControlPanelBottom

Reply:
Let's handle it with refactoring in next sprint

Reply:
Can you ask the POs for a refactoring issue?